nixos/modules/config/power-management.nix

```diff
@@ -71,7 +71,7 @@ in
 
     # FIXME: Implement powersave governor for sandy bridge or later Intel CPUs
     powerManagement.cpuFreqGovernor = mkDefault "ondemand";
-    powerManagement.scsiLinkPolicy = mkDefault "min_power";
+    powerManagement.scsiLinkPolicy = mkDefault "max_performance";
```

Comment:
I can do that. I wasn't quite sure how these defaults work. will update when I get home.